lib/bolt/config/modulepath.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'bolt/config'
+
+module Bolt
+  class Config
+    class Modulepath
+      BOLTLIB_PATH = File.expand_path('../../../bolt-modules', __dir__)
+      MODULES_PATH = File.expand_path('../../../modules', __dir__)
+
+      def initialize(modulepath)
```

Comment:
I think it would be better if this were a _little_ less purpose-built. Maybe methods like `boltlib`, `user_modules`, and `builtin_modules`? We could then make the constructor take those three pieces and have a factory method that 
 supplies the default locations for `boltlib` and `builtin_modules`. That might avoid the need for a PE fork altogether?